developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md

```diff
@@ -0,0 +1,155 @@
+---
+title: text2vec-jinaai
+sidebar_position: 13
+# tags: ['text2vec', 'text2vec-jinaai', 'jinaai']
+---
+
+
+## Overview
+
+Weaviate uses the `text2vec-jinaai` module to obtain vectors.
+- [JinaAI Embeddings](https://jina.ai/embeddings/)
+
+Key notes:
+
+- As it uses a third-party API, you will need an API key.
+- Enabling this module will enable the [`nearText` search operator](/developers/weaviate/api/graphql/search-operators.md#neartext).
+- The default model is `jina-embeddings-v2-base-en`.
+
+### Docker Compose file
+
+To use `text2vec-jinaai`, you must enable it in your Docker Compose file (`docker-compose.yml`). You can do so manually, or create one using the [Weaviate configuration tool](/developers/weaviate/installation/docker-compose.md#configurator).
+
+#### Parameters
+
+|Parameter|Required|Purpose|
+|:-|:-|:-|
+|`ENABLE_MODULES`|Required|The modules to enable. Include `text2vec-jinaai` to enable the module.|
```

Comment:
Consider 
|`ENABLE_MODULES`|Yes|

Are there other models? We should either list them or link to a list